recipes/react-router/app/lib/resolve-puck-path.server.ts

```diff
@@ -0,0 +1,13 @@
+export function resolvePuckPath(path: string = "") {
+  const url = new URL(path, "https://placeholder.com/");
```

Comment:
It's not clear that `placeholder.com` doesn't need to be changed by the user (in fact, I thought I would have to change it after generating).

I understand the technique here, but is there a cleaner way we could do this without requiring the placeholder string?

Reply:
I've added a comment saying it isn't important and set the origin as a default parameter value. Let me know if you want to change it to something else.